added asyncstorage to apollo client, but commented out till login get…

weilonglin opened the pull request:
…s merged

Commit:
added asyncstorage to apollo client, but commented out till login gets merged

weilonglin committed:
Merge branch 'development' into add-kids-and-auth

weilonglin committed:
Merge branch 'development' into add-kids-and-auth

DiegoOTdC reviewed (commented):
Man lot's of work! Great job! :D

weilonglin commented:
Added mutation for adding kids circle. Added Auth to apollo client, currently has a hardcoded token. Needs to be changed after the token expired. Awaiting login AsyncStorage setter.
